### PR TITLE
Updated Mongodb, Mongodb_exporter helm_version & Appverison

### PR DESCRIPTION
**Pull request by jshre898**

### CODE REVIEW

File: helm/values/mongodb/values.yaml [hunk 1]
```diff
@@ -766,19 +892,21 @@ externalAccess:
...
+    ## @param externalAccess.autoDiscovery.image.digest Init container auto-discovery image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
     ## @param externalAccess.autoDiscovery.image.pullPolicy Init container auto-discovery image pull policy
     ## @param externalAccess.autoDiscovery.image.pullSecrets Init container auto-discovery image pull secrets
     ##
     image:
       registry: docker.io
       repository: bitnami/kubectl
-      tag: 1.23.6-debian-10-r8
+      tag: 1.30.2-debian-12-r0
+      digest: ""
```
Comment (yuvraj-squareops1) [inline helm/values/mongodb/values.yaml:906]: Is there specific use of this digest here?

File: helm/values/mongodb/values.yaml [hunk 2]
```diff
@@ -789,27 +917,85 @@ externalAccess:
...
-    ## @param externalAccess.autoDiscovery.resources.requests Init container auto-discovery resource requests
+    ## @param externalAccess.autoDiscovery.resourcesPreset Set container resources according to one common preset (allowed values: none, nano, micro, small, medium, large, xlarge, 2xlarge). This is ignored if externalAccess.autoDiscovery.resources is set (externalAccess.autoDiscovery.resources is recommended for production).
+    ## More information: https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_resources.tpl#L15
     ##
-    resources:
-      ## Example:
-      ## limits:
-      ##    cpu: 100m
-      ##    memory: 128Mi
+    resourcesPreset: "nano"
```
Comment (yuvraj-squareops1) [inline helm/values/mongodb/values.yaml:928]: why you are using resourcesPreset here in place of resources?

File: helm/values/mongodb/values.yaml [hunk 3]
```diff
@@ -789,27 +917,85 @@ externalAccess:
...
+    ## @skip externalAccess.dnsCheck.image.tag Init container dns-check image tag (immutable tags are recommended)
+    ## @param externalAccess.dnsCheck.image.digest Init container dns-check image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+    ## @param externalAccess.dnsCheck.image.pullPolicy Init container dns-check image pull policy
+    ## @param externalAccess.dnsCheck.image.pullSecrets Init container dns-check image pull secrets
+    ##
+    image:
+      registry: docker.io
+      repository: bitnami/os-shell
+      tag: 12-debian-12-r23
+      digest: ""
```
Comment (yuvraj-squareops1) [inline helm/values/mongodb/values.yaml:956]: same here "digest", why?

File: README.md
Comment (yuvraj-squareops1) [inline README.md:1]: Still versions are not updated

File: examples/complete/aws/helm/exporter.yaml
Comment (yuvraj-squareops1) [inline examples/complete/aws/helm/exporter.yaml:1]: Do we have only changes in AWS?

Comment (jshre898) [reply examples/complete/aws/helm/exporter.yaml:1]: Yes.

File: helm/values/mongodb/values.yaml (hunk 1 above)
Comment (jshre898) [reply helm/values/mongodb/values.yaml:906]: No, It was added in current helm_chart. So added it and kept empty.

**Review by yuvraj-squareops1 — APPROVED** [review]

**Pull request merged**